src/Sentry/Protocol/Envelopes/EnvelopeItem.cs

```diff
@@ -320,9 +292,13 @@ await foreach (var curByte in stream.ReadAllBytesAsync(cancellationToken).Config
                 prevByte = curByte;
             }
 
-            return
-                Json.Parse(buffer.ToArray(), JsonExtensions.GetDictionaryOrNull)
-                ?? throw new InvalidOperationException("Envelope item header is malformed.");
+            var header = Json.Parse(buffer.ToArray(), JsonExtensions.GetDictionaryOrNull);
```

Comment:
Is this a refactor or something changed?

Reply:
Artifact between commits.  I'll undo.  Thanks